ntbs-service-tests/UnitTests/Models/NotificationTest.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using ntbs_service.Models;
+using Xunit;
+
+namespace ntbs_service_tests.UnitTests.Models
+{
+    public class NotificationTest
+    {
+
+        public static Notification TestNotification = new Notification {

```

Comment:
Changed it now - does this correctly separate them out with the class being recreated at the start of each test or does each test need a separate clone of the base class so that I'm not just adding fields to the same object each test?